packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panels_builder.dart

```diff
@@ -9,6 +9,12 @@ class KnowledgePanelsBuilder {
 
   List<Widget> build(KnowledgePanels knowledgePanels) {
     final List<Widget> rootPanelWidgets = <Widget>[];
+    if (knowledgePanels.panelIdToPanelMap['root'] == null) {
```

Comment:
can we merge these ? 

`if (knowledgePanels.panelIdToPanelMap['root'] == null || knowledgePanels.panelIdToPanelMap['root']!.elements == null) {
      return rootPanelWidgets;
    }
`